lib/ChatBot.jsx

```diff
@@ -611,6 +667,11 @@ class ChatBot extends Component {
     const previousStep = index > 0 ? renderedSteps[index - 1] : {};
     const previousSteps = index > 0 ? this.generateStepsById(renderedSteps.slice(0, index)) : {};
 
+    // .ValueStep's should not be rendered
```

Comment:
.ValueSteps* :neckbeard:

Reply:
I wanted to indicate that `.ValueStep` is something inside code while there is no `.ValueSteps` in the code. Perhaps I could write it as '.ValueStep's to communicate more?

Reply:
What I meant here is that in correct English it's `ValueSteps` not `ValueStep's` :p 
It's a minor comment, no worries.